File: 1_Variables_Y_Tipos_De_Datos/python/funcion_dia.py



def que_dia_es(dia):
    dias_de_la_semana = ['Lunes', 'Martes', 'Miercoles', 'Jueves', 'Viernes', 'Sabado', 'Domingo']
    return dias_de_la_semana[dia]

def que_mes_es(mes ):

    meses = ['enero','febrero','marzo','abril','mayo','junio','julio','agosto','septiembre','octubre','noviembre','diciembre']

    return meses[mes]

def que_fecha_es(dia,mes):
    a = que_dia_es(dia)
    b = que_mes_es(mes)
    return a,b
    #return que_dia_es(dia),que_mes_es(mes)


print(que_dia_es(1))
print( que_mes_es(3))
print(que_fecha_es(3,2))

#if __name__ == '__main__':   #por buena practica se hace asi, porque el "main" se refiere a que estas usando este modulo python como programa principal

File: 3_Funciones/java/src/Funcion_dia.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class Funcion_dia {

    public static void main(String[] args) {
        System.out.println(new Funcion_dia().queMesEs(1));
        System.out.println(new Funcion_dia().queDiaEs(5));
        System.out.println(new Funcion_dia().fecha(4,7));
    }

    public String queMesEs(int dia){
        ArrayList listaString = new ArrayList<String>(Arrays.asList("enero","febrero","marzo","abril",
                "mayo","junio","julio","agosto","septiembre","octubre","noviembre","diciembre"));
        return (String) listaString.get(dia);
    }

    public String queDiaEs(int mes){
        String  meses [] = new String[7];
        meses[0] = "Lunes";
        meses[1] = "Martes";
        meses[2] = "Miercoles";
        meses[3] = "Jueves";
        meses[4] = "Viernes";
        meses[5] = "Sabado";
        meses[6] = "Domingo";
        return meses[mes];
    }

    public List fecha(int dia, int mes){

        return new ArrayList(Arrays.asList(queDiaEs(dia),queMesEs(mes)));
    }





}
